Odata-docs/TOC.yml

```diff
@@ -324,6 +324,8 @@
         href: /odata/client/using-blazor-wasm-with-odata-client
       - name: Grouping and aggregation
         href: /odata/client/grouping-and-aggregation
+      - name: BulkUpdate Operations
```

Comment:
Should be "Bulk Update Operations" or "Bulk update operations".

Personally I favor sentence case though I tend to see a mix of both in Microsoft products docs https://learn.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-7.0&tabs=visual-studio